init-letsencrypt.sh

```diff
@@ -39,9 +39,9 @@ echo
 
 echo "### Deleting dummy certificate for $domains ..."
 docker-compose run --rm --entrypoint "\
-  rm -Rf /etc/letsencrypt/live/$domains && \
-  rm -Rf /etc/letsencrypt/archive/$domains && \
-  rm -Rf /etc/letsencrypt/renewal/$domains.conf" certbot
+  rm -rf /etc/letsencrypt/live/$domains && \
```

Comment:
I think you should leave the -Rf as it is. The -R is meant to be a recursive removal on directories and its sub-directories.